frontend/rust-lib/flowy-database2/src/services/field/type_options/checklist_type_option/checklist.rs

```diff
@@ -153,7 +156,7 @@ impl CellDataDecoder for ChecklistTypeOption {
 
   fn stringify_cell_data(&self, cell_data: <Self as TypeOption>::CellData) -> String {
     cell_data
-      .selected_options()
+      .options
       .into_iter()
       .map(|option| option.name)
       .collect::<Vec<_>>()
```

Comment:
I think it isn't much of a problem right now, but just food for thought. Checklist is the only type option that doesn't have a perfect string representation. Examples of other field types.

- text, number, url:  as is
- date, last_modified, created_at: formatted string March 2, 2024 16:38 -> March 2, 2024 16:52
- checkbox: yes or no
- single/multi_select: names of the selected options

I changed the implementation here from displaying only the completed items to displaying all the items, so that I ca test the sort. Previously, if a cell just had a single uncompleted item, the cell string would be "". Now, it will display that empty item as well. However, it doesn't say whether it's completed or not.